tests/test_init.py

```diff
@@ -41,7 +41,7 @@ def test_missing_backends(isolate_modules, param):
 
     try:
         delattr(pyhf.tensor, module_name)
-    except:
+    except:  # noqa: E722
```

Comment:
We might want to deal with these instead of the `noqa`

Reply:
these can be dealt with in another PR. It's just a matter of switching to `except Exception as err` and adding a `log.error(err)` to be safe.